submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
+  elem = document.createElement(elem);
+  elem.classList.add(className);
+  for (attrName in attrObj) {
+    elem.setAttribute(attrName, attrObj[attrName]);
+  }
+  return elem;
+};
+
+const createMenu = (data) => {
+  let menu = createElem("ul", { id: "menu", class: "menu" });
+  data.forEach(
+    (item, index) => { menu.appendChild(createMenuItem(data[index], index)) }
+  )
```

Comment:
You can use [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) + [append](https://developer.mozilla.org/en-US/docs/Web/API/Element/append)
p.s. I'm not sure that this code will work, but try to got my point. 
```suggestion
const menuItems = data.map((item, index) => createMenuItem(data[index], index))
menu.append(menuItems)

```